movieNormal.sql

```diff
@@ -0,0 +1,90 @@
+--Select all columns and rows from the movies table
+SELECT *
+FROM movies;
+
+--Select only the title and id of the first 10 rows
+SELECT title, movieid
+FROM movies LIMIT 10;
+
+--Find the movie with the id of 485
+SELECT *
+FROM movies
+WHERE movieid = 485;
+
+--Find the id (only that column) of the movie Made in America (1993)
+SELECT movieid
+FROM movies
+WHERE title LIKE 'Made in A%';
+
+--Find the first 10 sorted alphabetically
+SELECT *
+FROM movies
+ORDER BY title ASC
+LIMIT 10;
+
+--Find all movies from 2002
+SELECT *
+FROM movies
+WHERE title LIKE '%(2002)';
+
+--Find out what year the Godfather came out
+SELECT title
+From movies
+WHERE title LIKE '%Godfather%';
+
+--Without using joins find all the comedies
+SELECT *
+FROM movies
+Where genres LIKE 'Comedy%';
```

Comment:
This only works if comedy is at the front of the genres field.  Any other way will fail.